File: Canvas/app/src/main/java/com/example/canvas/MainActivity.java
package com.example.canvas;
import android.graphics.Bitmap;
import android.graphics.Canvas;
import android.graphics.Color;
import android.graphics.drawable.BitmapDrawable;
import android.graphics.drawable.ShapeDrawable;
import android.graphics.drawable.shapes.OvalShape;
import android.graphics.drawable.shapes.RectShape;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: Auto_complete_text_view/app/src/main/java/com/example/auto_complete_text_view/MainActivity.kt
package com.example.auto_complete_text_view
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.AutoCompleteTextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val countries=resources.getStringArray(R.array.countries)
        val arrayAdapter=ArrayAdapter(this,android.R.layout.simple_list_item_1,countries)
        val actv=findViewById<AutoCompleteTextView>(R.id.autoCompleteTextView)
        actv.setAdapter(arrayAdapter)
    }
}